packages-next/types/src/config/index.ts

```diff
@@ -22,7 +23,8 @@ export type KeystoneConfig = {
   db: DatabaseConfig;
   ui?: AdminUIConfig;
   server?: ServerConfig;
-  session?: SessionStrategy<any>;
+  session?: () => SessionStrategy<any>;
+  cloud?: CloudConfig;
```

Comment:
```suggestion
  session?: SessionStrategy<any>;
  cloud?: CloudConfig;
```

Reply:
Weird I don't remember doing that change... will do as suggested